addons/van_dml/example/README.md

```diff
@@ -4,9 +4,14 @@ This is a  demo showing the design pattern and the usage of the commands introdu
 
 To use van_dml, include the script right after vanJS (an ES6-module will come soon)
 ```HTML
-  <script src="van-0.11.10.nomodule-min.js"></script>
+  <script src="van-latest.nomodule-min.js"></script>
   <script src="van_dml.js"></script>
```

Comment:
Would it be better to use the CDN path in `README.md` as well to match the example in `index.html`? Also, this file https://github.com/vanjs-org/van/tree/main/addons/van_dml can be updated as well.